### ISSUE TITLE
waybar persistent workspaces update

### ISSUE THREAD

**Issue by mur1chan** [prasanthrangan/hyprdots#429]
As waybar updated their persistent_workspaces the workspaces are now always visible even though they are not used:
![image](https://github.com/prasanthrangan/hyprdots/assets/126001665/7806c94d-d9a6-4bfb-9bdc-636e741e5288)

it's just a config update i guess. i just wanted to mention the issue before everybody flips out :D 

i will look tomorrow at the waybar documentation and figure it out how to fix it if you aren't faster than me!

good night and i can't thank you enough for this great project

**Comment by T-Crypt** [comment]
@mur1chan 

Yep this was in update [0.9.23](https://github.com/Alexays/Waybar/releases/tag/0.9.23)

```
"hyprland/workspaces": {
    "disable-scroll": true,
    "active-only": false,   
    "all-outputs": false,
    "on-click": "activate",
    "sort-by-number": true,
    "persistent_workspaces": {
      "1": [],
      "2": [],
      "3": [],
      "4": [],
      "5": []
    }
  }
```

**Comment by T-Crypt** [comment]
Fixed in #432

**Comment by prasanthrangan** [comment]
> Fixed in #432 -- @mur1chan you can close this

I've merged this PR... so I'll close this!
